ui/app/mixins/policy-edit-controller.js

```diff
@@ -1,49 +0,0 @@
-import { inject as service } from '@ember/service';
```

Comment:
broke this logic up moved update logic to `policy-form` component, and delete logic to `policies/policy/edit` controller